lib/v8.js

```diff
@@ -368,7 +368,7 @@ class DefaultDeserializer extends Deserializer {
     }
     // Copy to an aligned buffer first.
     const buffer_copy = Buffer.allocUnsafe(byteLength);
-    copy(this.buffer, buffer_copy, 0, byteOffset, byteOffset + byteLength);
+    copy(this.buffer, buffer_copy, 0, byteOffset, byteLength);
```

Comment:
(I think) this.buffer.copy(...) would do a few more checks before going to the copy instrinsic... is that required here or is this fine?

Reply:
I think this is fine.